### PR TITLE
Allow more HTML files than just index.html

### PR DESCRIPTION
**Pull request by UDXS**

### CODE REVIEW

**Review by shreyasminocha — COMMENTED** [review]
I'm assuming the minified html will be outputted to `dist/index.html`?

**Comment by UDXS** [comment]
No, it outputs to multiple html files

**Comment by shreyasminocha** [comment]
Oh. Do you think there is enough use case for this in the context of the competition?

File: gulpfile.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ gulp.task('buildHTML', () => {
             js: paths.dist.js
         }))
         .pipe(minifyHTML())
-        .pipe(rename('index.html'))
+//        .pipe(rename('index.html'))
```
Comment (shreyasminocha) [inline gulpfile.js]: Please remove this commented out line

**Comment by shreyasminocha** [comment]
Just resolve the commented out line and we should be good to go.

File: gulpfile.js (hunk 1 above)
Comment (shreyasminocha) [inline gulpfile.js]: Please remove this commented out line and we'll be good to go.

**Review by shreyasminocha — APPROVED** [review]

**Pull request merged**